contrib/utxo-tools/utxo_to_sqlite.py

```diff
@@ -28,53 +33,9 @@
 }
 
 
-def read_varint(f):
-    """Equivalent of `ReadVarInt()` (see serialization module)."""
-    n = 0
-    while True:
-        dat = f.read(1)[0]
-        n = (n << 7) | (dat & 0x7f)
-        if (dat & 0x80) > 0:
-            n += 1
-        else:
-            return n
-
-
-def read_compactsize(f):
-    """Equivalent of `ReadCompactSize()` (see serialization module)."""
-    n = f.read(1)[0]
-    if n == 253:
-        n = int.from_bytes(f.read(2), "little")
-    elif n == 254:
-        n = int.from_bytes(f.read(4), "little")
-    elif n == 255:
-        n = int.from_bytes(f.read(8), "little")
-    return n
-
-
-def decompress_amount(x):
-    """Equivalent of `DecompressAmount()` (see compressor module)."""
-    if x == 0:
-        return 0
-    x -= 1
-    e = x % 10
-    x //= 10
-    n = 0
-    if e < 9:
-        d = (x % 9) + 1
-        x //= 9
-        n = x * 10 + d
-    else:
-        n = x + 1
-    while e > 0:
-        n *= 10
-        e -= 1
-    return n
-
-
 def decompress_script(f):
     """Equivalent of `DecompressScript()` (see compressor module)."""
-    size = read_varint(f)  # sizes 0-5 encode compressed script types
+    size = deser_varint(f)  # sizes 0-5 encode compressed script types
```

Comment:
👍, `read_varint` and `deser_varint` implementations match exactly (except for description, which we may move over there)